File: practice/java/Factorial.java
import java.lang.*;
import java.util.*;

class Factorial
{
	public static void main(String args[])
	{	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter a number : ");
		int n = sc.nextInt();
		int F=1;
		for(int i=1;i<=n;i++)
		{
			F=F*i;
		}
		System.out.println("The Factorial of the given number is : "+F);
	}
}

File: practice/c/factorial.c
#include<stdio.h>
int fact(int n)
{
	int f=1,i;
	for(i=1;i<=n;i++)
		f=f*i;
	return f;
}
int fact(int);
main()
{
	
	int n,f;
	printf("Enter a number :\n ");
	scanf("%d",&n);
	f=fact(n);
	printf("The factorial of the given number = %d",f);

}
